lib/services/zohoprojects.rb

```diff
@@ -0,0 +1,55 @@
+class Service::ZohoProjects < Service::Base
+  title 'Zoho Projects'
+
+  string :project_id,
+    :label => "Note: You need a ZohoProjects enterprise account to enable this integration. " \
```

Comment:
We should add a way to have a description or instructions instead of jamming everything into the first label.